components/Pages/Project/Review/ProgressBar.tsx

```diff
@@ -0,0 +1,23 @@
+interface ProgressBarProps {
+  currentStep: number;
+  numberOfItems: number;
+}
+
+export const ProgressBar = ({ currentStep, numberOfItems = 100 }: ProgressBarProps) => {
```

Comment:
It is not clear to me what a prop called number of items has in common to a ProgressBar, can it be renamed?